CookSmart/CookSmart/CSScaleView.m

```diff
@@ -207,10 +211,14 @@ static inline CGFloat getTargetContentOffset(CSScaleView *scaleView)
 
 static inline void setScrollViewOffset(CSScaleView *scaleView, CGPoint newContentOffset, BOOL cancelDeceleration)
 {
+  if (isnan(newContentOffset.y)) {
+    return;
+  }
```

Comment:
I had some weird crashes that I needed to fix with this. 🤷‍♂ Should be fine once this is converted to Swift

Reply:
Same here.